### ISSUE TITLE
Banner Official Gov site component  - Accessibility review

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#1302]
## Issue Description
We need to conduct a review of the accessibility guidance on USWDS for the Banner - Official Gov site component  https://designsystem.digital.gov/components/banner/  and compare it to the a11y guidance in the VA DS.  [(URL)](https://design.va.gov/components/banner/) 

## Tasks
- [x] Full review of accessibility guidance for the component on both USWDS and VA DS. 
- [ ] Identify and document the differences between the two 
- [ ] Make a recommendation on which guidance to use and include brief explanation on why (feedback can be in this ticket)
- [ ] Review final recommendation with the team (can be done during standup) 
- [ ] Add a11y guidance updates to the documentation ticket (if needed) 

## Acceptance Criteria
- [ ] Differences of the a11y guidance for the component has been documented (can be in this ticket)
- [ ] Recommendation has been shared with the team
- [ ] Documentation ticket has been updated with the revised text for a11y guidance (if needed)

**Comment by caw310** [comment]
Discussed that Sarah will expand on current banner accessibility guidance to include recommendations on why we use a banner.

**Comment by SarahKay8** [comment]
**We could extend our guidance by adding these a11y considerations for our Notification Banner**
Dismissable banner is used in place of toast notification. Avoid using a toast notification for critical alerts or long messages because they are timed and will disappear automatically making it difficult for people with various disabilities to get the entire message. An alert that disappears too quickly can lead to failure of the optional WCAG 2.0 success criterion 2.2.3 (AAA).
Alerts that need immediate attention should have focus shifted to them when the page loads.
Aria attributes should be added to give alerts accessible names. Those that need immediate attention should have a `role-alert` Other alerts should have a `role-region` and an `aria-label` set to "notice"

**Comment by SarahKay8** [comment]
@humancompanion-usds

**Comment by humancompanion-usds** [comment]
@SarahKay8 - This isn't a notification banner. USWDS chooses to call this component a "banner", which is terribly generic and thus causes some confusion. It's the grey bar that runs across the very top of all government web sites that communicates to the user that it is an official government web site.

**Comment by humancompanion-usds** [comment]
@caw310 - Let's discuss with Eli if we can get a11y-ops help to get a review of this component.

**Comment by caw310** [comment]
@humancompanion-usds , I reached out to Eli. Shared support isn't active anymore as the contract ran out.

**Comment by caw310** [comment]
This went through staging review
